src/protocol/http/src/test/groovy/org/apache/jmeter/protocol/http/control/DNSCacheManagerSpec.groovy

```diff
@@ -17,23 +17,26 @@
 
 package org.apache.jmeter.protocol.http.control
 
-import org.apache.jmeter.junit.spock.JMeterSpec
 import org.xbill.DNS.ExtendedResolver
 import org.xbill.DNS.ResolverConfig
 
 import spock.lang.IgnoreIf
+import spock.lang.Requires
+import spock.lang.Specification
 
-class DNSCacheManagerSpec extends JMeterSpec {
+class DNSCacheManagerSpec extends Specification {
 
-    private static final String[] VALID_DNS_SERVERS  = ResolverConfig.getCurrentConfig().servers()
+    private static final String[] VALID_DNS_SERVERS = ResolverConfig.getCurrentConfig().servers()
     private static final String INVALID_DNS_SERVER = "512.1.1.1"
+    private static final String VALID_HOSTNAME = "jmeter.apache.org"
 
-    static def localDNSResolverFailed() {
+    private static final boolean localDNSResolverOK = {
         try {
+            println "localDNSResolverOK"
```

Comment:
should this remain ?

Reply:
Nice catch. @ham1, would you please remove that?
```suggestion
```